docs/src/modules/components/HighlightedCode.js

```diff
@@ -1,30 +1,72 @@
+/* eslint-disable material-ui/no-hardcoded-labels */
```

Comment:
Why not translate 'Copied' and 'Copy'?

Reply:
Updated. But I am not sure how we can translate the markdown generated by `marked.js`.

Reply:
I think we should not translate this one. It is not consistent with the button generated by `markedjs`. We can revisit later when we have a better way.